packages/material-ui-lab/src/TreeView/TreeView.js

```diff
@@ -133,6 +135,12 @@ const TreeView = React.forwardRef(function TreeView(props, ref) {
     }
   };
 
+  const unfocus = (event, id) => {
+    if (!treeRef.current.contains(event.relatedTarget) && isFocused(id)) {
```

Comment:
This wouldn't work on IE 11. While relatedTarget is supported in IE11 for the native blur event, in react it isn't because react uses a different native event.